packages/react-components/src/localization/locales/es-ES/strings.json

```diff
@@ -338,4 +338,4 @@
     "leftNavButtonAriaLabel": "página anterior",
     "rightNavButtonAriaLabel": "página siguiente"
   }
-}
\ No newline at end of file
+}

```

Comment:
we need to run prettier over the localization PRs... This happens to me often as well

Reply:
it has massively bloated the PR I promise I didnt edit them 😅

Reply:
Yeah its when anyone merges in from main locally after a loc PR has gone in, prettier runs on the merge commit and goes - hey these should have new lines!

Reply:
(not sure why its not captured by prettier CI check... maybe its a git config thing?